static/app/api.tsx

```diff
@@ -448,7 +448,7 @@ export class Client {
       method,
       body,
       headers,
-      credentials: 'same-origin',
+      credentials: 'include',
```

Comment:
Are there any risks doing this?

Reply:
> Tells browsers to include credentials in both same- and cross-origin requests, and always use any credentials sent back in responses.

I think this is low risk as cookies are still domain matched. We could leak cookies to other `sentry.io` domains but that is currently possible with browser navigation.